.eslintrc.json

```diff
@@ -6,6 +6,7 @@
   "plugins": ["prettier"],
   "rules": {
     "prettier/prettier": "error",
+    "no-shadow": 0,
```

Comment:
What does this new setting?

Reply:
New eslint rule : https://eslint.org/docs/rules/no-shadow

Because the IoT Agents rely on callbacks and embedded callbacks like this:

```javascript
function start(callback) {
    // ... ETC
    function createConnection(callback) {
              callback();
    }
}
```

and ESLint doesn't like it when calling the embedded function with the same name as the outer function. It's a new rule so I've disabled it.

Reply:
NTC